docs/cmdline_migration.md

```diff
@@ -90,12 +90,13 @@ These HotSpot command-line options have equivalents in OpenJ9 that are not speci
 
 ## Creating compatible behavior
 
-You can set the following options to make OpenJ9 behave in the same way as Hotspot.
+You can set the following options to make OpenJ9 behave in the same way as HotSpot.
 
 | Option                                                           | Usage                                                                                                                                        |
 |------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
 | ![Start of content that applies only to Java 8 (LTS)](cr/java8.png) [`-Djava.lang.string.substring.nocopy=true`](djavalangstringsubstringnocopy.md) |  Avoid String sharing by String.substring(). ![End of content that applies only to Java 8](cr/java_close_lts.png) |
 | [`-Xnuma:none`](xnumanone.md)                                                     | Disable non-uniform memory architecture (NUMA) awareness.       |
+| [`-XXhandleSIGABRT`](xxhandlesigabrt.md) | Force handling of SIGABRT signals to be compatible with HotSpot |
```

Comment:
The PR still has old changes ... `-XX[+|-]handleSIGABRT` ... missing `:` and `H`. Guess ... after accepting my suggestions in the PR, you may have pushed from an outdated local branch. This would have undone the typo fixes.